src/screens/tasks.tsx

```diff
@@ -1,8 +1,105 @@
 import React from 'react'
-import { Switch, Route, useRouteMatch } from 'react-router-dom'
+import { Switch, Route, useRouteMatch, Link } from 'react-router-dom'
 // components
 import Layout from 'components/layout'
-import Task from 'domain/tasks/task'
+import TaskD from 'domain/tasks/task'
+//types
+import { Task } from 'types/tasks'
+import { MouseEventHandler } from 'react'
+
+import { isTaskExpired, getTaskByStatus } from 'utils/tasks'
+import arrayOfTasks from 'arrayOfTasks'
+
+interface myProps {
+  task: Task
+  icon: string
+}
+const TaskPresentation = ({ task, icon }: myProps): JSX.Element => {
+  return (
```

Comment:
This component should be moved to ```domain/tasks```. There we placed al componentes related with screen ```tasks```